mesh/mesh.hpp

```diff
@@ -2433,7 +2440,199 @@ class Mesh
 
 /** Overload operator<< for std::ostream and Mesh; valid also for the derived
     class ParMesh */
-std::ostream &operator<<(std::ostream &out, const Mesh &mesh);
+std::ostream &operator<<(std::ostream &os, const Mesh &mesh);
+
+/// @brief Print function for Mesh::FaceInformation.
+std::ostream& operator<<(std::ostream &os, const Mesh::FaceInformation& info);
+
+
+// Class containing a minimal description of a part (a subset of the elements)
```

Comment:
Doxygen style could be used.

Reply:
Okay, I'll convert them to doxygen. One potential concern is that the text comments are meant to be read in order they appear in the file and I'm not sure if Doxygen will preserve this order. I guess out-of-order doxygen comments are better than no comments in the doxygen output. 😄